Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableAssetRef.h

```diff
@@ -14,20 +14,20 @@
 #include <AzCore/Asset/AssetCommon.h>
 #include <AzFramework/Spawnable/Spawnable.h>
 
-namespace ScriptCanvas::Nodeables::Spawning
+namespace AzFramework
 {
-    //! A wrapper around Spawnable that can be used by Script Canvas
-    struct SpawnableAsset final
+    //! A wrapper around Spawnable asset that can be used by Script Canvas and Lua
+    struct SpawnableAssetRef final
```

Comment:
I've renamed it to follow same naming standard as ScriptEventsAssetRef which uses similar technique to expose asset field in Script Canvas